src/layouts/CoreLayout/index.js

```diff
@@ -8,10 +10,12 @@ require('../../styles/core.scss');
 const CoreLayout = ({ children }) => (
 
     <div>
+        <Header />
         <div className={Classes.mainContainer}>
             {children}
         </div>
         <Footer />
+        <Camera />
```

Comment:
Should this really be a `<Kegbot>` component?  It would probably house the `<Camera>` component, but it will also likely have more to add (and control).